### PR TITLE
Added text into Futures History Table and Spot History Table.

### PR DESCRIPTION
**Pull request by sebfk**
Fix to ticket #1043 

## Description
Added text into the tables in Future History Table and Spot History Table, if no trades have been made.

## How Has This Been Tested?
Ran it in local enviorment, and tested it with both logged into L2 and not logged into L2. Also tried doing some testnet transaction and tested if the text would then disappear.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "printWidth": 100,
   "semi": true,
   "singleQuote": true,
-  "trailingComma": "es5"
+  "trailingComma": "es5",
+  "endOfLine": "crlf"
```
Comment (koredefashokun) [inline .prettierrc:7]: I think we should leave this as the default (`lf`). Is there a specific reason you had to change it to `crlf`?

File: .eslintrc [hunk 2]
```diff
@@ -3,7 +3,11 @@
 	"plugins": ["prettier"],
 	"rules": {
 		"react/react-in-jsx-scope": "off",
-		"prettier/prettier": "error",
+		"prettier/prettier": "error", [
+		{
+		"endOfLine: "auto"
+		}
+		],
```
Comment (koredefashokun) [inline .eslintrc:10]: Just wanted to check if there was a reason for changing the `endOfLine` config.

File: .prettierrc (hunk 1 above)
Comment (LeifuChen) [reply .prettierrc:7]: @koredefashokun the dev is using windows so it requires adding this to make the lint work properly. however, there is no need to commit to the PR. I already asked the dev to reset this file.

File: .eslintrc (hunk 2 above)
Comment (LeifuChen) [reply .eslintrc:10]: @koredefashokun same reason. it's the setting for the dev who is working with windows but there is no need to commit to the PR.  @sebfk please reset this file in your commit.

**Review by LeifuChen — CHANGES_REQUESTED** [review]
@sebfk Don't include `.eslintrc` and `.prettierrc` in your commit. Keep them locally.

File: sections/dashboard/SpotHistoryTable/SpotHistoryTable.tsx [hunk 3]
```diff
@@ -251,6 +262,22 @@ const StyledText = styled.div`
...
+	justify-items: center;
+	grid-gap: 10px;
+	color: ${(props) => props.theme.colors.selectedTheme.button.text};
+	font-size: 20px;
+	font-family: ${(props) => props.theme.fonts.bold};
+	div {
+		text-decoration: underline;
+		font-size: 16px;
+		font-family: ${(props) => props.theme.fonts.regular};
+	}
```
Comment (LeifuChen) [inline sections/dashboard/SpotHistoryTable/SpotHistoryTable.tsx:278]: Please add `cursor: pointer;` into `div {...}`.
Same css change for the `FutureHistoryTable.tsx` (it's not your problem but it's good that you can fix it in this PR).

**Review by LeifuChen — CHANGES_REQUESTED** [review]
@sebfk One css change is requested in both the future history table and the spot history table.

File: translations/en.json [hunk 4]
```diff
@@ -538,7 +538,9 @@
 				"date-time": "Date/Time",
 				"from": "From",
 				"to": "To",
-				"usd-value": "USD Value"
+				"usd-value": "USD Value",
+				"no-trade-history": "You have no trades yet",
```
Comment (platschi) [inline translations/en.json:542]: Can you add a dot to the end of this sentence, so it's in line with the other one? :)

**Review by LeifuChen — APPROVED** [review]
@sebfk  Thanks for the contribution. Looks good to me.

**Pull request merged**